sandworm/lib/web.mlx

```diff
@@ -0,0 +1,237 @@
+          in
+
+            (<details class_="flex flex-col gap-2">
+              <summary class_="font-medium text-lg">
+                "dune " (JSX.string date)
+              </summary>
+              <table class_="w-full pb-4">
+                <thead>
+                  <tr>
+                    <th>"File name"</th>
+                    <th>"OS"</th>
+                    <th>"Arch"</th>
+                    <th>"Commit"</th>
+                    <th>"Certificate"</th>
+                  </tr>
+                </thead>
+                <tbody>
+                  (JSX.list files)
+                </tbody>
+              </table>
+            </details>)
+      )
+      |> JSX.list
+    )
+    </section>
+  </section>
+
+let manual_installation ~url:_ ~data () =  
+  <section class_="flex flex-col gap-2">
+    <h2 id="manual-installation" class_="text-3xl font-black pb-2"> "Manual Installation" </h2>
+    <p>
+      "In this section you'll find instructions to manually install the Dune
+      Developer Preview, and verify the installation. This is recommended for
+      advanced users, or if you want so understand where the binaries come from."
+    </p>
+
+    <h2 class_="text-2xl font-medium pt-4 pb-2"> "Featured Downloads" </h2>
+    <p class_="pb-2">
+      "This is the latest release of the Dune Developer Preview. See the "
+      <a class_="text-[var(--accent)]" href="#release-history">"release history"</a>
+      " section for all past releases."
+    </p>
+    <ul class_="flex flex-row flex-wrap gap-5 justify-center items-center align-center">
+      (let bundle: Metadata.Bundle.t = data |> List.hd in
+      bundle.targets
+       |> List.map installation_target
+       |> JSX.list)
+    </ul>
+
+    <h2 class_="text-2xl font-medium pt-4 pb-2"> "Installing the binary" </h2>
+    <p>
+      "After downloading a binary release of Dune, make it executable and place it somewhere reachable by your PATH:"
+      <pre>
+        "
+$ chmod u+x ./dune
+$ sudo mv ./dune /usr/local/bin/dune
+        "
+      </pre>
+      "You can verify your installation by running:"
+      <pre>
+      {|
+$ dune --version
+"Dune Developer Preview: build , git revision cc260345db57ab639db6363b2dc89072a1492832"
+|}
+      </pre>
+    </p>
+
+    <h2 class_="text-2xl font-medium pt-4 pb-2"> "Verifying the Dune binary" </h2>
+    <p>
+      <span>"To ensure trust in the binary distribution, we generate a build certificate associated with the Github Actions pipeline where the binaries are built. Once you download this certificate, you can use the "</span>
+      <code>"gh"</code>
+      <span>"tool to verify it with the following command: "</span>
+      <pre>
+"
+$ gh attestation verify ./dune -R tarides/dune-binary-distribution --bundle attestation.jsonl
+"
+      </pre>
+    </p>
+    <release_history data />
+  </section>
+
+let page ~url data =
+  <html>
+    <head>
+      <meta charset="utf-8" />
+      <title> "Dune Developer Preview" </title>
```

Comment:
What do you think about extracting the title in a variable? It would make it easier to change in the future (as at some point it will just be Dune Distribution). But maybe it's a problem for later.